Tests/SentryTests/SentryViewPhotographerTests.swift

```diff
@@ -210,6 +237,44 @@ class SentryViewPhotographerTests: XCTestCase {
         assertColor(pixel1, .green)
     }
     
+    func testNotMaskingLabelInsideclippedViewHiddenByAnOpaqueExternalView() throws {
+        let topView = UIView(frame: CGRect(x: 25, y: 0, width: 25, height: 25))
+        topView.backgroundColor = .green
+        
+        let label1 = UILabel(frame: CGRect(x: 0, y: 0, width: 50, height: 25))
+        label1.text = "Test"
+        label1.textColor = .black
+        
+        let parentView = UIView(frame: CGRect(x: 0, y: 0, width: 50, height: 25))
+        parentView.addSubview(label1)
+        parentView.clipsToBounds = true
```

Comment:
`m`: If we want to test `clipsToBounds`, shouldn't the `label1` be bigger than the `parentView`. Currently, they are the same size, and `clipsToBounds` shouldn't make a difference. Or is there another reason to use it?

I think if you set the background color of the parentView to red and increase the height of the label1 to 30, no red color should be visible, cause the label and the topView should be drawn over the parentView.

Reply:
Not for this test. I just need to parent to clip its bounds because it changes the flow of masking.